browser/ui/webui/brave_webui_source.cc

```diff
@@ -363,6 +365,48 @@ void CustomizeWebUIHTMLSource(const std::string &name, content::WebUIDataSource*
         { "whyBraveRewardsDesc2", IDS_BRAVE_UI_WHY_BRAVE_REWARDS_DESC_2 },
         { "yourWallet", IDS_BRAVE_UI_YOUR_WALLET }
       }
+    }, {
+      std::string("bravesync"), {
```

Comment:
I'll look at this because we shouldn't just be dumping all of these in this file